static/src/javascripts/projects/commercial/modules/cmp/cmp.js

```diff
@@ -88,6 +89,12 @@ const isInCommercialConsentGlobalBannerTest = (): boolean =>
 const isInEU = (): boolean =>
     (getCookie('GU_geo_continent') || 'OTHER').toUpperCase() === 'EU';
 
+const isInAU = (): boolean =>
+    (getFromStorage() || 'OTHER').toUpperCase() === 'AU' ||
```

Comment:
Cache common expression in a constant to avoid redundant calls.

Reply:
Done!